webclient/components/1time-1row-mField/mental-status-exam/db/client-side/initialize-on-first-load.vue

```diff
@@ -0,0 +1,27 @@
+<template>
+  <div>
+    <initializeMentalStatusExamComponent />
+  </div>
+</template>
+
+<script>
+import initializeMentalStatusExamComponent from '@/components/1time-1row-mField/mental-status-exam/db/client-side/static-data/insert-into-master-of-search-phrases-ct.vue'
```

Comment:
Parent is initialize-on-first-load
The child should be:
step1-initilize-search-phrase-of-mental-status-exam
step2-get-latest-date-from-db-server